### PR TITLE
Implementation of a Adaboost Algorithm in Python,Java,C and C++

### PR DESCRIPTION
**Pull request by abhishekmpote**
This pull request introduces an implementation of the AdaBoost algorithm in various programming languages: Python, C++, C, and Java. Each implementation follows the same logic and functionality, allowing users to apply the AdaBoost algorithm for binary classification and multiclass classification tasks.

Issue #1074 

👨‍💻 Changes proposed

1. Added code to fit the AdaBoost classifier in Python.
2. Created a AdaBoost Classifier class with methods to fit the classifier and make predictions in C++.
3. Defined the necessary structs and functions to implement AdaBoost classifier in C.
4. Created a AdaBoost Classifier class with methods for fitting and predicting in Java.

✔️ Check List (Check all the applicable boxes) 

- [✅] My code follows the code style of this project.
- [✅] This PR does not contain plagiarized content.
- [✅] The title of my pull request is a short description of the requested changes.

📄 Note to reviewers
Please review the changes in each implementation and provide feedback on the code structure, algorithm implementation, and any potential improvements.
Thank you for your time and valuable feedback.

Best regards,
Abhishek Pote

image
image
image
cpp

### CODE REVIEW

File: Java/Machine Learning/Adaboost/AdaBoostClassifier.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import java.util.ArrayList;
```
Comment (Kumar-laxmi) [inline Java/Machine Learning/Adaboost/AdaBoostClassifier.java:1]: Place the program directly in ```Machine Learning``` folder

File: Python/Machine Learning/Adaboost/AdaBoostClassifier.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+'''
```
Comment (Kumar-laxmi) [inline Python/Machine Learning/Adaboost/AdaBoostClassifier.py:1]: Place the program directly in ```Machine Learning``` folder

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Comment by Kumar-laxmi** [comment]
@abhishekmpote Kindly resolve the CODEFACTOR issues and provide the screen shot of output and not the Code

**Review by zugzwang03 — APPROVED** [review]

**Review by zugzwang03 — APPROVED** [review]

**Review by Arun9739 — APPROVED** [review]

**Review by 072arushi — APPROVED** [review]

**Review by zugzwang03 — APPROVED** [review]

**Review by Kumar-laxmi — COMMENTED** [review]
Is this code generated by ChatGPT because the code structure seems to be generated by an AI

**Review by Arun9739 — CHANGES_REQUESTED** [review]
@abhishekmpote The code looks plagarized and output also incorrect I think

**Pull request closed without merging**